### PR TITLE
R: few updates and some new packages

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Weekend updates and some new ports for R.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`? (for new ports)
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by barracuda156** [comment]
@herbygillot This is good to merge

**Comment by ryandesign** [comment]
> This is good to merge

Except for:

```
port test R-multinma:
Warning: R-multinma is configured to build no architecture-specific files, but installed Mach-O files built for the following archs:
Warning: x86_64:
Warning:   /opt/local/Library/Frameworks/R.framework/Versions/4.4/Resources/library/multinma/libs/multinma.so
```

**Comment by barracuda156** [comment]
> > This is good to merge
> 
> Except for:
> 
> ```
> port test R-multinma:
> Warning: R-multinma is configured to build no architecture-specific files, but installed Mach-O files built for the following archs:
> Warning: x86_64:
> Warning:   /opt/local/Library/Frameworks/R.framework/Versions/4.4/Resources/library/multinma/libs/multinma.so
> ```

@ryandesign Sorry, this is accidental. Of course it does build arch-specific files. I will fix it now. Thank you.

**Comment by barracuda156** [comment]
@ryandesign @herbygillot Fixed.

File: R/R-LBI/Portfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+revision            0
+categories-append   math
+maintainers         nomaintainer
+license             GPL-3
+description         Likelihood-Based Inference
+long_description    {*}${description}
+checksums           rmd160  af06dacd109d990b1a67b2ccdc761fd01f984495 \
+                    sha256  a506aa12c47625be05c0636447727b4360fd31d25db53f7e1cd664cb5cf72e86 \
+                    size    8956
+supported_archs     noarch
```
[the remark this answers is not in the document]
Comment (barracuda156) [reply R/R-LBI/Portfile:16]: @jmroot Sure. Is there a way to know for sure if installed files are identical across OS versions? (Besides manual comparison, of course.)

[the remark this answers is not in the document]
Comment (jmroot) [reply R/R-LBI/Portfile:16]: Automated comparison? :)

I haven't come across a noarch R-* port yet that differs materially. But I guess a build system could theoretically do just about anything?

The build timestamp gets embedded in a lot of files, so we would have to at least set `SOURCE_DATE_EPOCH` to get bit-for-bit identical files. A useful tool to use for comparisons is `diffoscope`.

**Pull request merged**